sessionctx/variable/session.go

```diff
@@ -1294,6 +1298,12 @@ func (s *SessionVars) SetSystemVar(name string, val string) error {
 		s.FoundInPlanCache = TiDBOptOn(val)
 	case TiDBEnableCollectExecutionInfo:
 		config.GetGlobalConfig().EnableCollectExecutionInfo = TiDBOptOn(val)
+	case SQLSelectLimit:
+		result, err := strconv.ParseUint(val, 10, 64)
+		if err != nil {
+			return errors.Trace(err)
+		}
+		s.SelectLimit = result
```

Comment:
Better to raise a warning that SQL Plan Management would not work now when setting this variable?

Reply:
A warning returned to MySQL Client or A warning log ? which way do you think is ok ?